src/Configuration/src/Kubernetes.ServiceBinding/LegacyConnectorsPostProcessors.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+namespace Steeltoe.Configuration.Kubernetes.ServiceBinding;
+
+internal sealed class RabbitMQLegacyConnectorPostProcessor : IConfigurationPostProcessor
+{
+    internal const string BindingTypeKey = "rabbitmq";
```

Comment:
do you see a use for these `const` values outside these classes, or should they be private?

Reply:
Used in unit tests